### ISSUE TITLE
[Feature Request] Improved / more usage friendly checkpoint loader

### ISSUE THREAD

**Issue by Darkflo264** [comfyanonymous/ComfyUI#332]
Hi all!

I tried to find a solution myself, but sadly failed terrible at this.

i think I have read about something similar before, but was not able to find it again.

i would really love a improved checkpoint leader with a button to open a open-file window where a checkpoint directly can be selected. A bit Like at the open-image node.

This current „infinity list from hell“ drives me crazy. Meanwhile I do have 1+ TB of checkpoints, about 300-400 files. I think I can’t be the only one who collects them like coins….

A simple list without search or filter is not great in this case. Copying just a few files to the linked checkpoint directory ain’t cool too.

Even better would be a filter or search field.
Or, if it is possible, some selection menu like for nodes? 

I Imagine this like, directories in the checkpoint dir are shown as categories, files in these are shown below them. So we could at least sort them into categories like „realistic“, „anime, „whatever“.

**Comment by ronmurphy** [comment]
i agree, that's why i asked if they could make the loader, or a version of this, that can load up the preview images that the civita automatic1111 downloads... clicking on a long file name that can't fit in the widget because it is named  "SomOrangeAbyssalSuperReduxMix00.69.888056_hiRes.safetensor" is crazy, an image box that loads up the preview png, if it exists, would be useful.   or make a standard grid array like how fuaomatic1111 has, as it's own gui loader.

https://github.com/comfyanonymous/ComfyUI/issues/331

**Comment by trojblue** [comment]
Maybe  implementing a gallery node would be really helpful in that sense. It could be used to export arbitary strings like checkpoint names or even image file names, then fed into other nodes
